src/plugins/gmx/gmxplugin.cpp

```diff
@@ -83,6 +83,26 @@ GmxPlugin::GmxPlugin()
 {
 }
 
+bool GmxPlugin::checkIfViewsDefined(LayerIterator iterator)
```

Comment:
Since this function makes no use of `GmxPlugin` state, it can be a local static function:
```c++
static bool checkIfViewsDefined(const Map *map)
```
Also since there is no performance gain in re-using iterator, I would suggest just passing in the map and having a local iterator in the function.